docs/book/src/developer/tilt.md

```diff
@@ -13,9 +13,16 @@ workflow that offers easy deployments and rapid iterative builds.
 1. [kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md) standalone
    (`kubectl kustomize` does not work because it is missing some features of kustomize v3)
 1. [Tilt](https://docs.tilt.dev/install.html) v0.12.0 or newer
+1. [envsubst](https://github.com/a8m/envsubst) v1.1.0 or similar to handle clusterctl var replacement.
```

Comment:
If we really are unable to use `drone/envsubst` as a binary here, I suggest we document that `drone/envsubst` and `a8m/envsubst` have similar behavior because the docs in https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#variables call out our usage of `drone/envsubst`. 
This inconsistency may be confusing to users.